recipes/obspy/run_test.py

```diff
@@ -0,0 +1,9 @@
+import matplotlib
+matplotlib.use("AGG")
+from obspy.core import run_tests
+errors = run_tests(report=True, hostname="conda-forge", verbosity=2)
```

Comment:
@megies if the full test suit is "heavy weight" you might not want to run it here. (I am assuming you do run it in your CIs, right?)

Most of the time we are OK with simpler "is the package OK" kind of tests. Like imports and command line checks.